code/datums/tutorial/marine/role_specific/hospital_corpsman_basic.dm

```diff
@@ -0,0 +1,1090 @@
+
+	UnregisterSignal(tutorial_mob, COMSIG_HUMAN_EQUIPPED_ITEM)
+
+	TUTORIAL_ATOM_FROM_TRACKING(/obj/item/clothing/head/helmet/marine/medic, helmet)
+	remove_highlight(helmet)
+
+	manual_message_to_player("While not as detailed as a Health Analyzer scan, your <b>HealthMate HUD</b> will display a healthbar over the head of any injured Marine, which will indicate their general condition.")
+	manual_message_to_player("When someone does not have a healthbar over their head, or when their healthbar disappears, that means they are <b>Fully Healthy</b>.")
+	manual_message_to_player("As you progress through the following sections of the tutorial, pay attention to how the Dummys healthbar changes with different injuries.")
+
+	register_tutorial_step(CALLBACK(src, PROC_REF(brute_tutorial)))
+
+/datum/tutorial/marine/role_specific/hospital_corpsman_basic/proc/brute_tutorial()
+
+	manual_message_to_player("<b>Section 1: Basic Damage Treatment</b>")
+	manual_message_to_player("<b>Section 1.1: Brute Damage</b>")
+	manual_message_to_player("There are two main types of damage a Marine can sustain through injuries, each with a different method of treatment.")
+	manual_message_to_player("The first kind of damage is <b>Brute</b>, the most common kind. It represents physical trauma from things like punches, weapons, or guns.")
+	register_tutorial_step(CALLBACK(src, PROC_REF(brute_tutorial_2)))
+
+/datum/tutorial/marine/role_specific/hospital_corpsman_basic/proc/brute_tutorial_2()
+	SIGNAL_HANDLER
+
+	marine_dummy = new(loc_from_corner(2,2))
+	add_to_tracking_atoms(marine_dummy)
+	arm_equipment(marine_dummy, /datum/equipment_preset/uscm/tutorial_rifleman/mrdummy)
+	marine_dummy.apply_damage(5, BRUTE, "chest")
+	manual_message_to_player("The Dummy has taken some kind of brute damage. Stand next to them, and click on their body with your <b>Health Analyzer</b> in hand to scan them.")
+	add_highlight(marine_dummy, COLOR_GREEN)
+	update_objective("Click on the Dummy with your Health Analyzer")
+	RegisterSignal(marine_dummy, COMSIG_LIVING_HEALTH_ANALYZED, PROC_REF(scanner_3))
+
+/datum/tutorial/marine/role_specific/hospital_corpsman_basic/proc/scanner_3(datum/source, mob/living/carbon/human/attacked_mob)
+	SIGNAL_HANDLER
+
+	if(attacked_mob == tutorial_mob)
+		return
+
+
+	UnregisterSignal(marine_dummy, COMSIG_LIVING_HEALTH_ANALYZED)
+
+	manual_message_to_player("Good. By looking at the Health Analyzer interface, we can see they have 5 brute damage on their chest.")
+	manual_message_to_player("A chemical called <b>Bicaridine</b> is used to heal brute damage over time.")
+	manual_message_to_player("<b>Bicaridine</b> is primarily given in pill form.")
+	register_tutorial_step(CALLBACK(src, PROC_REF(brute_tutorial_3_pre)))
+
+/datum/tutorial/marine/role_specific/hospital_corpsman_basic/proc/brute_tutorial_3_pre(obj/item/storage/pill_bottle)
+	SIGNAL_HANDLER
+
+	handle_pill_bottle(marine_dummy, "Bicaridine", "Bi", "11", /obj/item/reagent_container/pill/bicaridine)
```

Comment:
if you feed yourself multiple pills in a row you have to click through a billion lines of repeated dialog